screens/Login/Login.js

```diff
@@ -18,17 +43,37 @@ class Login extends React.Component {
           <Input
             label="EMAIL"
             labelStyle={{ color: colors.primary }}
+            onChangeText={text => this.setState({ ...this.state, email: text })}
             placeholder="Email"
           />
           <Input
             placeholder="Password"
             labelStyle={{ color: colors.primary }}
             label="PASSWORD"
+            onChangeText={text =>
+              this.setState({ ...this.state, password: text })
+            }
             secureTextEntry
           />
         </View>
         <Text style={styles.forgotPassword}>Forgot your password?</Text>
-        <Button buttonStyle={style.loginButton} title="LOG IN" />
+        <Button
+          buttonStyle={style.loginButton}
+          onPress={this.login}
+          title="LOG IN"
+        />
+
+        <Button
+          buttonStyle={style.loginButton}
+          onPress={this.logout}
+          title="LOG OUT"
+        />
+
+        <Button
+          buttonStyle={style.loginButton}
+          onPress={this.getUser}
+          title="CHECK TOKEN"
+        />
```

Comment:
check token and logout buttons temporary to show token storage with login. recommend keeping in until we can test with real backend